files/en-us/web/accessibility/aria/aria_live_regions/index.html

```diff
@@ -189,20 +191,20 @@ <h3 id="Advanced_use_case_Clock"><span class="mw-headline" id="Use_Case:_Clock">
 }, 60000);
 </pre>
 
-<p>The first time the function executes, the entirety of the string that is added will be announced. On subsequent calls, only the parts of the content that changed compared to the previous content will be announced. For instance, when the clock changes from "17:33" to "17:34", assistive technologies will only announce "4", which won't be very useful to users.</p>
+<p>The first time the function executes, the entirety of the string that is added will be announced. On subsequent calls, only the parts of the content that changed compared to the previous content will be announced. For instance, when the clock changes from <code>"17:33"</code> to <code>"17:34"</code>, assistive technologies will only announce <samp>"4"</samp>, which won't be very useful to users.</p>
 
 <p>One way around this would be to first clear the contents of the live region, and then inject the new content. However, this can sometimes be unreliable, as it's dependent on the exact timing of these two updates.</p>
 
-<p><code>aria-atomic="true"</code> ensures that each time the live region is updated, the entirety of the content is announced in full (e.g. "Time: 17:34").</p>
+<p><code>aria-atomic="true"</code> ensures that each time the live region is updated, the entirety of the content is announced in full (e.g. <samp>"Time: 17:34"</samp>).</p>
```

Comment:
```suggestion
<p><code>aria-atomic="true"</code> ensures that each time the live region is updated, the entirety of the content is announced in full (e.g. "<samp>Time: 17:34</samp>").</p>
```

Moving the `samp` element to wrap only what AT might announce.